cranelift/codegen/src/ir/immediates.rs

```diff
@@ -74,10 +74,27 @@ impl Imm64 {
         self.0
     }
 
+    /// Mask this immediate to the given power-of-two bit width.
+    pub(crate) fn mask_to_width(&mut self, bit_width: u32) {
```

Comment:
To fit with the other operator methods (e.g. `wrapping_neg`), can we do functional style here (`fn mask_to_width(self, bit_width: u32) -> Self`)?